include/hammer_peg.h

```diff
@@ -0,0 +1,14 @@
+#ifndef GUARD_HAMMER_PEG_H
+#define GUARD_HAMMER_PEG_H
+
+#include "global.h"
+#include "data.h"
+#include "functions.h"
+#include "object.h"
+#include "kirby.h"
+
+struct Object2 *CreateHammerPeg(struct Object *, u8);
```

Comment:
```suggestion
void *CreateHammerPeg(struct Object *, u8);
```
They have to return `void *` since all these `Create*` functions are in a function array.